sw/device/lib/testing/keymgr_testutils.h

```diff
@@ -30,4 +30,15 @@ void keymgr_testutils_generate_identity(const dif_keymgr_t *keymgr);
 void keymgr_testutils_generate_versioned_key(
     const dif_keymgr_t *keymgr, const dif_keymgr_versioned_key_params_t params);
 
+/**
+ * Issues a keymgr disable and wait for it to complete
+ */
```

Comment:
nit: it seems like this file is missing @param & @return comments.

Reply:
Updated, thanks